model/env.go

```diff
@@ -49,3 +51,25 @@ func (em *EnvVarMap) ToEnvList() []corev1.EnvVar {
 
 	return envList
 }
+
+// ToJSON converts the EnvVarMap to a JSON object represented as a
+// []byte
+func (em *EnvVarMap) ToJSON() ([]byte, error) {
+	b, err := json.Marshal(em)
+	if err != nil {
+		return []byte(nil), err
+	}
+
+	return b, nil
```

Comment:
`json.Marshal()` returns the same thing as the parent function here. I would either wrap the error if there is one or just return the marshal call directly.

Reply:
🤔 Yeah, you're right. I think for some reason I was thinking just returning the call to `json.Marshal` directly was in conflict with the project style? Idk, good catch.

Reply:
Might even just want to get rid of this method and call `json.Marshal` directly at the call site? I'm unsure.

Reply:
Yeah, that was one of the reasons I originally didn't make this method, but I am totally fine keeping it.